src/dumbo/src/tcp/endpoint.rs

```diff
@@ -316,6 +316,16 @@ fn parse_request_bytes(byte_stream: &[u8], callback: fn(Request) -> Response) ->
     match request {
         Ok(request) => callback(request),
         Err(e) => match e {
+            RequestError::Overflow => build_response(
+                Version::default(),
+                StatusCode::BadRequest,
+                Body::new(e.to_string()),
+            ),
+            RequestError::Underflow => build_response(
+                Version::default(),
+                StatusCode::BadRequest,
+                Body::new(e.to_string()),
+            ),
```

Comment:
nit: since this seems unreachable now, I would move this arm at the end of this match, right before the `UnsupportedHeader` one (I wouldn't though replace `build_response()` with `unreachable!()`since maybe `try_from()` eventually will be able to return `Underflow` too and it would be pretty easy to forget updating here with it).

Reply:
Done.